frontend/src/app/core/setup/globals/global-listeners.ts

```diff
@@ -84,6 +84,17 @@ export function initializeGlobalListeners():void {
       performAnchorHijacking(evt, linkElement);
     });
 
+  // Listen for 'zenModeToggled' event to toggle Zen Mode styling on the body.
+  // Adds 'zen-mode' class if active; removes it if not.
+  window.addEventListener('zenModeToggled', (event:CustomEvent) => {
+    const isActive = event.detail.active;
+    if (isActive) {
+      document.body.classList.add('zen-mode');
+    } else {
+      document.body.classList.remove('zen-mode');
```

Comment:
💡 You can simplify the whole method by using [toggle](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle)

```
window.addEventListener('zenModeToggled', (event:CustomEvent) => {
  document.body.classList.toggle('zen-mode', event.detail.active)
}
```

Reply:
Hi @ulferts thanks for good hintwise, much better now 👍